rayon-core/src/lib.rs

```diff
@@ -560,7 +560,7 @@ impl<S> ThreadPoolBuilder<S> {
     /// to true, however, workers will prefer to execute in a
     /// *breadth-first* fashion -- that is, they will search for jobs at
     /// the *bottom* of their local deque. (At present, workers *always*
-    /// steal from the bottom of other worker's deques, regardless of
+    /// steal from the bottom of other worker's dequeues, regardless of
```

Comment:
This is not a typo -- just the plural of "deque" that you left on the line above.

Reply:
The plural possessive should be updated though...
```suggestion
    /// steal from the bottom of other workers' deques, regardless of
```